README.md

```diff
@@ -60,229 +35,230 @@ After you're finished please remove all the comments and instructions!
+- Slack: [Chandan Gupta](@ChandanGupta)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- FUTURE FEATURES -->
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-> Describe 1 - 3 features you will add to the project.
+- The features I will add to the project are :
 
-- [ ] **[new_feature_1]**
-- [ ] **[new_feature_2]**
-- [ ] **[new_feature_3]**
+  - [ ] **[Improve mobile responsiveness]**
+  - [ ] **[Integration with a back-end database]**
+  - [ ] **[User authentication and account creation]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- CONTRIBUTING -->
 
 ## 🤝 Contributing <a name="contributing"></a>
 
-Contributions, issues, and feature requests are welcome!
+- Understand the project's goals: Before I start contributing to a project, it's important to understand its goals, scope, and target audience. Read the project's documentation and review its codebase to get a sense of what it does and how it works.
+
+- Review existing issues: Check the project's issues page to see if there are any open issues that you can help with. Look for issues that match my skillset or interests, and read through the discussion to understand the problem and any proposed solutions.
+
+- Create a new issue: If you I find an existing issue that matches your contribution, consider creating a new one. sure about the problem or feature request clearly and provide any relevant details or context.
 
-Feel free to check the [issues page](../../issues/).
+- Fork the repository: If I ready to start contributing code, fork the project's repository on GitHub. This will create a copy of the project that I can work on independently.
+
+- Submit a pull request: When I am ready to submit my changes back to the main project, create a pull request on GitHub. Sure about the changes clearly and provide any relevant details or context.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- SUPPORT -->
 
 ## ⭐️ Show your support <a name="support"></a>
 
-> Write a message to encourage readers to support your project
-
-If you like this project...
+- If you like this project, please consider showing your support by starring the repository on GitHub! Your star helps to increase the project's visibility and encourages others to use and contribute to it. Additionally, if you find any issues or have any feature requests, please don't hesitate to submit them to the project's issues page. Your feedback and contributions are greatly appreciated and help to make the project better for everyone. Thank you for your support!
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
+- I would like to express my gratitude to the following individuals and resources who have inspired and contributed to this project:
 
```

Comment:
- [ ] Kindly give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) in your acknowledgment, as required by the [Creative Commons License](https://creativecommons.org/licenses/).